concepts/05 UI Components/DataGrid/15 Columns/40 Customize Cells/3 Customize Row Apperance.md

```diff
@@ -0,0 +1,147 @@
+To customize the existing row layout, implement the [onRowPrepared](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onRowPrepared) event handler. This handler allows you to customize the existing row layout after it is generated. If a customization is not immediately applied, use the [repaintRows(rowIndexes)](/Documentation/ApiReference/UI_Components/dxDataGrid/Methods/#repaintRowsrowIndexes) method to repaint DataGrid rows.
```

Comment:
You use the word "layout" in the first sentence. I think we better switch it to "appearance". The topic is about appearance, and you don't really change the layout in the example. "Layout" is more about the position of elements relative to each other, but the example only changes colors.